libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm

```diff
@@ -21,10 +21,10 @@ CAM1_RELAY_ON,0
 CAM1_SERVO_OFF,1000
 CAM1_TYPE,2
 CAN_D1_UC_OPTION,10
-CAN_D1_UC_SRV_BM,1824
+CAN_D1_UC_SRV_BM,22784
```

Comment:
Shouldn't this be 24352?

Reply:
yes I forgot to update this but it should be 16160
![image](https://github.com/user-attachments/assets/6c7a2802-fcd4-4048-992e-9bb18fecd233)

Reply:
Need Servo 15 checked, not Servo 14